javascript-loops/main.js

```diff
@@ -0,0 +1,67 @@
+/* exported getNumbersToTen,
+            getEvenNumbersToTwenty,
+            repeatWord,
+            logEachCharacter,
+            doubleAll,
+            getKeys,
+            getValues
+ */
+function getNumbersToTen() {
+  var number = [];
+  var currentNumber = 1;
+  while (currentNumber <= 10) {
+    number.push(currentNumber);
+    currentNumber++;
+  }
+  return number;
+}
+
+function getEvenNumbersToTwenty() {
+  var evenNumbers = [];
+  var currentNumber = 2;
+  while (currentNumber <= 20) {
+    evenNumbers.push(currentNumber);
+    currentNumber += 2;
+  }
+  return evenNumbers;
+}
+
+function repeatWord(word, times) {
+  var count = 1;
+  var repeated = '';
+  while (count <= times) {
+    repeated += word;
+    count += 1;
+  }
+  return repeated;
+}
+
+function logEachCharacter(string) {
+  for (var i = 0; i < string.length; i++) {
+    console.log(string[i]);
+  }
+}
+
+function doubleAll(numbers) {
+  var doubled = [];
+  for (var i = 0; i < numbers.length; i++) {
+    doubled.push(numbers[i] *= 2);
```

Comment:
Oh no, this is bad news because you are changing the original array. You want to use the values of the array given as an argument, but you never want to change the original values.